hack/verify-golint.sh

```diff
@@ -44,12 +44,16 @@ export IFS=$'\n'
 # NOTE: when "go list -e ./..." is run within GOPATH, it turns the k8s.io/kubernetes
 # as the prefix, however if we run it outside it returns the full path of the file
 # with a leading underscore. We'll need to support both scenarios for all_packages.
-all_packages=(
-  $(go list -e ./... | egrep -v "/(third_party|vendor|staging/src/k8s.io/client-go/pkg|generated|clientset_generated)" | sed -e 's|^k8s.io/kubernetes/||' -e "s|^_\(${KUBE_ROOT}/\)\{0,1\}||")
-)
-failing_packages=(
-  $(cat $failure_file)
-)
+all_packages=()
```

Comment:
```
In ./hack/verify-golint.sh line 48:
  $(go list -e ./... | egrep -v "/(third_party|vendor|staging/src/k8s.io/client-go/pkg|generated|clientset_generated)" | sed -e 's|^k8s.io/kubernetes/||' -e "s|^_\(${KUBE_ROOT}/\)\{0,1\}||")
  ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
                       ^---^ SC2196: egrep is non-standard and deprecated. Use grep -E instead.
Ref: https://www.shellcheck.net/wiki/SC2207
```